r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -1304,3 +1304,53 @@ test_that("dminutes, dhours, ddays, dweeks, dmonths, dyears", {
     ignore_attr = TRUE
   )
 })
+
+test_that("dseconds, dmilliseconds, dmicroseconds, dnanoseconds, dpicoseconds", {
+  example_d <- tibble(x = c(1:10, NA))
+  date_to_add <- ymd("2009-08-03", tz = "America/Chicago")
```

Comment:
```suggestion
  date_to_add <- ymd("2009-08-03", tz = "Pacific/Marquesas")
```

If we don't need a specific feature of Chicago's timezone here, we should use our default "unusual" timezone of "Pacific/Marquesas" so that we have a low probability of this being a developer's timezone